pyanaconda/anaconda_log.py

```diff
@@ -32,6 +32,10 @@
 STDOUT_FORMAT = "%(asctime)s %(message)s"
 DATE_FORMAT = "%H:%M:%S"
 
+# the Anaconda log uses structured logging
+ANACONDA_ENTRY_FORMAT = "%(asctime)s,%(msecs)03d %(levelname)s %(log_prefix)s: %(message)s"
```

Comment:
Couldn't we have a boot option `inst.debug`, that would enable more detailed structured messages? 
It looks like there are a lot of attributes we could log: 
https://docs.python.org/3/library/logging.html#logrecord-attributes